examples/language/lib/shorthands/enums.dart

```diff
@@ -0,0 +1,15 @@
+// Enum example
```

Comment:
Since the introductory text for this snippet makes this clear, I think you can drop this comment:

```suggestion
```